cmake/script/Coverage.cmake

```diff
@@ -5,6 +5,7 @@
 include(${CMAKE_CURRENT_LIST_DIR}/CoverageInclude.cmake)
 
 set(functional_test_runner test/functional/test_runner.py)
+list(APPEND functional_test_runner --timeout-factor=8)  # Required, because coverage builds are slower
```

Comment:
I've noticed the same -- https://github.com/hebasto/bitcoin/issues/341#issuecomment-2310510852:
> However, on my machine, I had to add `--timeout-factor=4`.